WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/PrepublishingHomeViewModel.kt

```diff
@@ -205,16 +205,29 @@ class PrepublishingHomeViewModel @Inject constructor(
 
     private fun MutableList<PrepublishingHomeItemUiState>.showSocialItem() {
         // TODO in other PR: use actual data, for now just using fake data
+//        add(
```

Comment:
❓ Should we uncomment/remove this block in this PR?

Reply:
I left it here to be able to easily toggle between the fake data scenarios. This will be removed when we implement the actual data fetching.